### DISCUSSION TITLE
Digest authorisation method?

### DISCUSSION THREAD

**Discussion by sfreyer** [thunderclient/thunder-client-support#654] [Ideas]
I just saw the tweet where Thunder was praised ("I no longer need Postman"). VS Code is definitely my preferred go-to-tool. My need is to use _digest_ as authorisation method. Could this easily be implemented in next version of Thunder?

**Comment by rangav** [comment]
@sfreyer thanks for feedback,

We have feature request for this already 
https://github.com/rangav/thunder-client-support/issues/413

Currently it is in our roadmap

I haven’t used digest auth before, if you have any implementation link in node, will look into it.
